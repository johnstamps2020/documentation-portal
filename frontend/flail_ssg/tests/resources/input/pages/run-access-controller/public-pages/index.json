{
  "$schema": "/frontend/page-schema.json",
  "title": "Automated redirect",
  "class": "",
  "template": "redirect",
  "items": [
    {
      "label": "cloudNotPublic",
      "page": "cloudProductsNotPublic"
    },
    {
      "label": "cloudPublic",
      "page": "cloudProductsPublic"
    },
    {
      "label": "_redirect",
      "link": "/cloudProducts/dobson"
    }
  ]
}
