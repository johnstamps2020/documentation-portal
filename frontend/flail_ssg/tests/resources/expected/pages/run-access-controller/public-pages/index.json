{
  "$schema": "/frontend/page-schema.json",
  "title": "Automated redirect",
  "class": "",
  "template": "redirect",
  "items": [
    {
      "label": "cloudNotPublic",
      "page": "cloudProductsNotPublic",
      "public": false,
      "internal": false
    },
    {
      "label": "cloudPublic",
      "page": "cloudProductsPublic",
      "public": true,
      "internal": false
    },
    {
      "label": "_redirect",
      "link": "/cloudProducts/dobson"
    }
  ],
  "public": true,
  "internal": false
}
